tpm2/src/unseal/main.c

```diff
@@ -165,6 +187,12 @@ int main(int argc, char **argv)
                   "default output filename (%s) already exists ... exiting",
                   outPath);
         free(default_outPath);
+        kmyth_clear_and_free(outputData, outputSize);
+        if (authString != NULL)
```

Comment:
If we moved up the clear of authString and ownerAuthPasswd in the non-error case, then we wouldn't need to clear them here. We would still need to the clear and free of outputData, though.